### ISSUE TITLE
FEATURE REQUEST: Is there a way to bulk checkout consumables/accessories?

### ISSUE THREAD

**Issue by ABagOfFritos** [grokability/snipe-it#6928]
#### Expected Behavior (or desired behavior if a feature request)

We have boxes upon boxes of USB sticks, we want to use this tool to track them along with our other assets.  When we add the assets, we can put the total amount we have.  When we check out to a user, we expect to be able to input how many of these get checked out.

-----

#### Actual Behavior

Only a single USB can be checked out.  To check out 20 USBs to a single person, we would have to perform 20 checkouts.

-----

#### Please confirm you have done the following before posting your bug report:

- [/ ] I have enabled debug mode (not a bug, uncertain if this would be a feature request)
- [x ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? 
Fresh install
- Version of Snipe-IT you're running
Unknown
- Version of PHP you're running
Unknown
- Version of MySQL/MariaDB you're running
Unknown
- What OS and web server you're running Snipe-IT on
Ubuntu Server 18.04
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
Unknown
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
No errors
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
No errors
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
Demo also has no bulk checkout option - beginning to lean more towards this being a feature request.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by ABagOfFritos** [comment]
Bumping and changing this to a feature request

**Comment by klsweat** [comment]
This is still an issue? is there a solution to this?

**Comment by RivoHenfri** [comment]
there is no way solution, hopefully, anyone can help. I've got the same problem too.

**Comment by Player7up** [comment]
I took would like to see this option. In my case I have a number of training tablets (Kindle Fire HD 10's) that when they do training would request a number of them at a time.

**Comment by thiago-schmid** [comment]
We need this feature because: Our marketing team has an event to attend, they need to check out 1000 T-Shirts, but they only way to accomplish this is to manually checkout 1000 items.
I was impressed that this was not a default functionality since we have accessories, consumables that are only reasonable to checkout in bulk.

**Comment by webdawg** [comment]
It is pretty rough not having this feature.  Importing items, and checking them out to locations does not seem to work....even being able to edit and check out would be nice.

**Comment by Gregged** [comment]
It seems Snipe-IT have deliberately put this limitation as an annoyance, I cannot see how this would have been a miss, if you have 100000 items, without bulk checkouts it means you have to checkout 100000 times; worse yet, there are numerous case numbers without any responses. I was interested in the software BUT this one annoyance makes it utterly useless.

**Comment by IMS-IT-Subscriptions** [comment]
Bump I want it

**Comment by snipeituser1** [comment]
Bump I need it

**Comment by kaytearthurs** [comment]
I want it and need it pls

**Comment by darkpixel** [comment]
Great.  Stalebot closed it and no one with authority is re-opening it...I *really* hate Stalebot.

**Comment by snipe** [comment]
It’s on a very long list of feature requests.

**Comment by darkpixel** [comment]
I totally get it, not if feature request tickets will take a while to complete, stalebot is annoying and useless. ;)

**Comment by colins1019** [comment]
Would love this feature, we currently have 2200 asset that need to be checked out  after importing 👍

**Comment by HCurren** [comment]
While I was very excited to see that bulk check-in is being included in the upcoming Version 6, I would still think a bulk check-out feature would be immensely valuable. My company uses Snipe-IT to track tools used by it's technicians. They have to check-out about 40 tools before each job, and check them back in on arrival. It seems to be one of the parts of the job that they dread the most, due to the numerous steps required. As it is currently, the steps required for this process are as follows;

1. Remove tool from tool storage
2. Hit tab to select "search by asset number" box
3. use scanner to scan barcode
4. put both items down and fill out the forms 
5. Click check in
6. Put tool into tool bag to prep for departure.
7. Repeat steps 1 through 6 forty times. 


We would love to see a simple option to fill out the Check-out fields first (Who is the item being checked out to, when will they be checked back in, notes, etc), and then scan all of the items sequentially, reducing the steps to the following;

1. Click "Check-out assets" (or whatever it would be called)
2. fill out the forms
3. Remove tool from storage
4. Scan
5. Put tool in bag
6. Repeat steps 3 through 5 forty times. 

While it may only reduce the number of steps by 50% (*only* lol), the amount of physical effort and transitioning it would require would save us hours per week, and I do not believe I am exaggerating about this.

**Comment by nealhamiltonjr** [comment]
So, after four years and numerous request for something that is simple to implement in the code this still remains; it appears you don't care about your users. I'm sure a lot of us would have paid a small fee to help since we know you don't work for free but you don't even seem interested in listening.

**Comment by snipe** [comment]
@nealhamiltonjr Yes, we don’t care about our users - that’s why we release new versions with improvements every single month. And we’ve done that - for free - for 9 years. 🙄

Our priority was shipping version 6, and now that version 6 is out, we can start working on a lot of things we’ve wanted to work on. This feature is one of many features people have asked for. We ship a lot of features all the time - we just haven’t gotten to this one yet. 

You seriously took the time to write this nasty message. What did you think it would accomplish? Do you think acting like an ungrateful jerk is going to get this feature built faster? Fortunately for you, we don’t withhold features because one person is being a jerk. We know this feature is important to a percentage of our users. As I have said before, it’s on the list - but leaving obnoxious comments does exactly nothing to move the ball forward here.

Attitudes like this are why maintainers take their software closed-source.

**Comment by nealhamiltonjr** [comment]
It's been four years and you guys don't even acknowledge the users, this
has been asked several times and no response. I'm not the only one that has
said what I said..get over it! If you don't like what I said, change your
behavior.

**Comment by snipe** [comment]
I spend half my day on Github and Discord, so to say I don’t acknowledge them is simply bullshit. 

We have had 10k+ issues since the project started - it’s simply not possible to respond to every one. 

> I'm not the only one that has said what I said..get over it! If you don't like what I said, change your behavior.

Or I can just block you from the repo and you can find other people to provide free labor for you.

**Comment by snipe** [comment]
I’m locking this convo now. To everyone except neal, please do know that this is still very much on our radar, and we appreciate your continued patience and support.

**Comment by snipe** [comment]
This was added ages ago
